docbook/reference/en/en-US/modules/StringConverter.xml

```diff
@@ -665,5 +665,152 @@ public static class TestResource {
+@GET
+public String pathMultiSeparator(@PathParam("p") @Separator("-") List&lt;String&gt; ss); 
+</programlisting>
+
+   <para>
+   will turn
+   </para>
+   
+<programlisting>
+List&lt;String&gt; list = new ArrayList&lt;String&gt;();
+list.add("abc");
+list.add("xyz");
+proxy.pathMultiSeparator(list);
+</programlisting>
+
+  <para>
+  and "path/separator/multi/{p}" into ".../path/separator/multi/abc-xyz". On the server side,
+  the RESTEasy runtime will turn "abc-xyz" back into a list consisting of elements "abc" and "xyz" for
+  </para>
+  
+<programlisting>
+@Path("path/separator/multi/{p}")
+@GET
+public String pathMultiSeparator(@PathParam("p") @Separator("-") List&lt;String&gt; ss) {
+   StringBuffer sb = new StringBuffer();
+   for (String s : ss) {
+      sb.append(s);
+      sb.append("|");
+   }
+   return sb.toString();
+}
+</programlisting>
+  
+   <para>
+   which will return "abc|xyz|".
+   </para>
+   
+   <para>
+   In fact, the value of the <classname>Separator</classname> annotations may be a more general regular
+   expression, which is passed to <methodname>String.split()</methodname>. For example, "[-,;]" tells the
+   server side to break up a string using either "-", ",", or ";". On the client side, a string will be created
+   using the first element, "-" in this case.
+   </para>
+   
+   <para>
+   If a parameter is annotated with <classname>@Separator</classname> with no value, then the default value is
+   </para>
+   
+   <itemizedlist>
+      <listitem>"," for a <classname>@HeaderParam</classname>, <classname>@MatrixParam</classname>,
+      <classname>@PathParam</classname>, or <classname>@QueryParam</classname>, and
+      </listitem> 
+      
+      <listitem>
+      "-" for a <classname>@CookieParam</classname>.
+      </listitem>
+   </itemizedlist>
+   
+   <para>
+   These <classname>ParamConverter</classname>s are meant to be fairly general, but there are a number of
+   restrictions:
+   </para>
+   
+   <orderedlist>
+      <listitem>They don't handle nested <classname>Collections</classname> or arrays. That is,
+      <classname>List&lt;String&gt;</classname> and <code>String[]</code> are OK, but
+      <classname>List&lt;List&lt;String&gt;&gt;</classname> and <code>String[][]</code> are not. 
+      </listitem>
+      
+      <listitem>
+      The regular expression used in <classname>Separator</classname> must match the regular expression
+<programlisting>
+"\\p{Punct}|\\[\\p{Punct}+\\]"
+</programlisting>
+
+      That is, it must be either a single instance of a punctuation symbol, i.e., a symbol in the set
+<programlisting>
+!"#$%&amp;'()*+,-./:;&lt;=&gt;?@[\]^_`{|}~
+</programlisting>
+
+      or a class of punctuation symbols like "[-,;]".
+      </listitem>
+      
+      <listitem>
+      For either of these <classname>ParamConverter</classname>s to be available for a given parameter, that
+      parameter must be annotated with <classname>@Separator</classname>.
+      </listitem>

```

Comment:
According to what you've said above, the annotation `@Separator` can also target a method `@Target({ElementType.METHOD, ElementType.PARAMETER})`. 
Can you please clarify, what does this annoation mean when just annotating a method ?

Reply:
@NicoNes, it was a careless cut-and-paste. Thanks, again.